cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -998,6 +1009,17 @@ rules:
   - delete
   - get
   - update
+- apiGroups:
```

Comment:
I initially thought the `cronjobs` RBAC settings should be defined in the `rook-ceph-system` `Role`, however I found that trying to access `cronjobs` without the permissions being provided on the `ClusterRole` lead to access errors. Could this also be the case for `jobs` as well, which is why the permissions are specified in `ClusterRole`?

Reply:
Ok, I'm not seeing why it should be on the clusterrole instead of the role. Both of them have a binding to the rook-ceph-system service account. Since it's working, and to be consistent with the jobs, we can keep it on the ClusterRole. Then you shouldn't need to add it to the role, right?